longest_common_prefix.py

```diff
@@ -0,0 +1,51 @@
+'''Write a function to find the longest common prefix string amongst an array of strings.
+If there is no common prefix, return an empty string "".
+Constraints:
+    1 <= strs.length <= 200
+    0 <= strs[i].length <= 200
+    strs[i] consists of only lower-case English letters.
+
+The linke for this question in leetcode: https://leetcode.com/problems/longest-common-prefix/
+'''
+
+
+'''This function takes list of strings as input and its job is to ensure that the input strings are within the constraints'''
+def is_valid_list(list_strings):
+    new_list_strings = []
+    if len(list_strings) >= 1 and len(list_strings) <= 200:
+        for string in list_strings:
+            if len(string)>=0 and len(string)<=200 and string.isalpha():
+                new_list_strings.append(string.lower())
+            else:
+                return False
+        list_strings = new_list_strings
+        return True,list_strings
+    return False
```

Comment:
dont ever make a function have different number of return values 
you use return True, list_string inside if => returning 2 values 
and in some cases u return single value